### PR TITLE
shellshare 1.0.1 (new formula)

### PR DESCRIPTION
**Pull request by arthar360**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/shellshare.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class Shellshare < Formula
+  desc "LIVE TERMINAL BROADCAST"
```
Comment (fxcoudert) [inline Formula/shellshare.rb]: Don't write in all uppercase please

File: Formula/shellshare.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  homepage "http://shellshare.net"
```
Comment (fxcoudert) [inline Formula/shellshare.rb]: Audit says `  * The URL http://shellshare.net could use HTTPS rather than HTTP`

File: Formula/shellshare.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  url "https://github.com/vitorbaptista/shellshare/archive/v1.0.0.tar.gz"
+  sha256 "72bd49928982611ae44aac4a308dc6054d9c8ebad87644bd5eb68ff0bb704ce3"
+  def install
+    bin.install "public/bin/shellshare"
+  end
+  test do
```
Comment (fxcoudert) [inline Formula/shellshare.rb]: Please provide some testing of the functionality.

File: Formula/shellshare.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+class Shellshare < Formula
+  desc "LIVE TERMINAL BROADCAST"
+  homepage "http://shellshare.net"
+  url "https://github.com/vitorbaptista/shellshare/archive/v1.0.0.tar.gz"
+  sha256 "72bd49928982611ae44aac4a308dc6054d9c8ebad87644bd5eb68ff0bb704ce3"
+  def install
```
Comment (fxcoudert) [inline Formula/shellshare.rb]: Please list requirements, such as Python (`depends_on :python if MacOS.version <= :snow_leopard`).

**Review by fxcoudert — DISMISSED** [review]

**Review by arthar360 — COMMENTED** [review]
Removed upper cases, Added dependencies and tests.

File: Formula/shellshare.rb [hunk 5]
```diff
@@ -0,0 +1,13 @@
+class Shellshare < Formula
+  desc "Live Terminal Broadcast"
+  homepage "https://shellshare.net"
+  url "https://github.com/vitorbaptista/shellshare/archive/v1.0.0.tar.gz"
+  sha256 "72bd49928982611ae44aac4a308dc6054d9c8ebad87644bd5eb68ff0bb704ce3"
```
Comment (MikeMcQuaid) [inline Formula/shellshare.rb]: Add a space between this line and the next.

File: Formula/shellshare.rb [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+  depends_on :python
```
Comment (MikeMcQuaid) [inline Formula/shellshare.rb]: Add a space between this line and the next.

File: Formula/shellshare.rb [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+  def install
+    bin.install "public/bin/shellshare"
+  end
```
Comment (MikeMcQuaid) [inline Formula/shellshare.rb]: Add a space between this line and the next.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

**Comment by arthar360** [comment]
Done. Please have a look.

**Review by arthar360 — COMMENTED** [review]
Added spaces between lines

File: Formula/shellshare.rb [hunk 8]
```diff
@@ -0,0 +1,16 @@
+class Shellshare < Formula
+  desc "Live Terminal Broadcast"
+  homepage "https://shellshare.net"
+  url "https://github.com/vitorbaptista/shellshare/archive/v1.0.0.tar.gz"
+  sha256 "72bd49928982611ae44aac4a308dc6054d9c8ebad87644bd5eb68ff0bb704ce3"
+
+  depends_on :python
```
Comment (MikeMcQuaid) [inline Formula/shellshare.rb]: Use `depends_on :python if MacOS.version <= :snow_leopard` to just depend on Python 2.7.

Comment (MikeMcQuaid) [inline Formula/shellshare.rb]: Add `bottle :unneeded` above this with a line space before and after.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

File: Formula/shellshare.rb [hunk 9]
```diff
@@ -0,0 +1,16 @@
+class Shellshare < Formula
+  desc "Live Terminal Broadcast"
+  homepage "https://shellshare.net"
+  url "https://github.com/vitorbaptista/shellshare/archive/v1.0.0.tar.gz"
+  sha256 "72bd49928982611ae44aac4a308dc6054d9c8ebad87644bd5eb68ff0bb704ce3"
```
Comment (vitorbaptista) [inline Formula/shellshare.rb]: I just released version 1.0.1, so please use that instead: https://github.com/vitorbaptista/shellshare/releases/tag/v1.0.1

**Review by arthar360 — COMMENTED** [review]
Added bottle :unneeded and MacOS.version <= :snow_leopard for Python 2.7. @vitorbaptista : Updated to v1.0.1

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks for your first contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by arthar360** [comment]
Thank you @MikeMcQuaid :)

**Comment by pwalsh** [comment]
Very happy to see shellshare added!

**Pull request merged**